### PR TITLE
Application icon created and Save/Save As features implemented

### PR DESCRIPTION
**Pull request by rohan-singh1**
- Code refactored for consistency.
- Application icons created and configured for macOS.
- Save and Save As features implemented for CSV file type.

### CODE REVIEW

File: src/csvexporter.cpp [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            processedRow.chop(1);           // Removing final comma
+            processedRow.append("\n");
+            processedOutputString.append(processedRow);
+        }
+        processedOutputString.chop(1);      // Removing final line break
+        outStream << processedOutputString;
+        isSuccessful = true;
+    }
+    file.close();
+    return isSuccessful;
```
Comment (leblancPDX) [inline src/csvexporter.cpp:45]: Minor: This function is pretty long. You might consider breaking it into some subroutines to improve readability.

**Review by leblancPDX — APPROVED** [review]
LGTM

File: src/csvexporter.cpp (hunk 1 above)
Comment (rohan-singh1) [reply src/csvexporter.cpp:30]: Done. The data processing before writing to the file is now done by a different function. Thanks for the feedback.

File: src/mainwindow.cpp [hunk 2]
```diff
@@ -119,6 +121,27 @@ void MainWindow::on_actionHelp_triggered()
 
 void MainWindow::on_actionAbout_triggered()
 {
+    QString result;
+    result.append("<b><u>About Kommedek</u></b><br>");
```
Comment (rohan-singh1) [inline src/mainwindow.cpp:125]: Typo

File: src/mainwindow.cpp [hunk 3]
```diff
@@ -119,6 +121,27 @@ void MainWindow::on_actionHelp_triggered()
...
+    result.append("Version 01.05<br><br>");
+    result.append("<b>Dear Proffesor:</b><br><br>");
```
Comment (rohan-singh1) [inline src/mainwindow.cpp:127]: Typo

**Pull request merged**